compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/ArgumentsGenerationUtils.kt

```diff
@@ -422,17 +425,23 @@ fun StatementGenerator.generateSamConversionForValueArgumentsIfRequired(call: Ca
                 "$originalDescriptor has ${originalDescriptor.typeParameters}"
     }
 
+    val partialSam = context.languageVersionSettings.supportsFeature(LanguageFeature.SamConversionPerArgument)
```

Comment:
Minor: give it some meaningful name, like `isPartialSamConversionSupported` or something.